asapdiscovery/data/utils.py

```diff
@@ -220,7 +224,11 @@ def cdd_to_schema(cdd_csv, out_json=None, out_csv=None, achiral=False):
 
     ## Get rid of the </> signs, since we really only need the values to sort
     ##  enantiomer pairs
-    pic50_key = "ProteaseAssay_Fluorescence_Dose-Response_Weizmann: Avg pIC50"
+    pic50_key = (
+        "pIC50"
+        if "pIC50" in df.columns
+        else "ProteaseAssay_Fluorescence_Dose-Response_Weizmann: Avg pIC50"
```

Comment:
Since the names of these CDD Protocols will change from assay to assay (`ProteaseAssay_Fluorescence_Dose-Response_Weizmann`), can we move these to some file-level `dict` for easier updating, or perhaps a YAML file we can modify as needed?

Reply:
Also, we don't want to use the `Avg pIC50` since this is the average of multiple measurements. We want to use the _best quality_ single measurement IC50 and lower/upper CIs.
![image](https://user-images.githubusercontent.com/3656088/217713085-9df5afe8-2138-4f6f-ad7f-120f5047955a.png)